core/services/workflows/workflow.go

```diff
@@ -11,37 +11,45 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
-type Capability struct {
+// yamlStep is the yaml parsed representation of a step in a workflow.
+type yamlStep struct {
```

Comment:
Hmmm... I'm a bit less keen on calling this `yamlStep`, since we could add other tags to this at a later date and therefore opt into different formats. However, that may be a case of premature optimisation.

In my latest iteration I called this `capabilityDefinition` so wdyt about `stepDefinition`?